projectile.el

```diff
@@ -1438,6 +1444,14 @@ If PROJECT is not specified the command acts on the current project."
     (dolist (buffer project-buffers)
       (funcall action buffer))))
 
+(defun projectile-process-current-project-buffers-current (action)
```

Comment:
I think you probably didn't want to add the second "current". :-)

Reply:
Ah, I read first the code and then your explanation - I get the point of the function, but I still think this name is quite confusing.

Reply:
Hi! I agree that the name is a bit off. However, projectile-process-current-project-buffers is already taken, so I had to add the second current. I wanted to base the name on projectile-process-current-project-buffers, because the behaviour of my function and that one should be similar. You can of course propose a better name, and I would be happy to use that instead.